lib/correios/cep/parser.rb

```diff
@@ -15,6 +15,10 @@ class Parser
 
       def address(xml)
         doc = Ox.parse(xml)
+        
+        errors = get_errors(doc)
+        return errors unless errors.nil?
```

Comment:
`errors` variable is a `String`. It is expected `Parser#address` method returns a `Hash`.